bundles/org.openhab.binding.sony/README-SCALAR.md

```diff
@@ -0,0 +1,783 @@
+| st_registrationdate-{src}       | R (5)      | String         | The storage registration date                         |
+| st_systemareacapacitymb-{src}   | R (5)      | Number         | The storage system capacity (in MB)                   |
+| st_timesectofinalize-{src}      | R (5)      | Number         | The time (in seconds) to finalize                     |
+| st_timesectogetcontents-{src}   | R (5)      | Number         | The time (in seconds) to get contents                 |
+| st_type-{src}                   | R (5)      | String         | The storage type                                      |
+| st_uri-{src}                    | R (5)      | String         | The storage URI                                       |
+| st_usbdevicetype-{src}          | R (5)      | String         | The storage USB device type                           |
+| st_volumelabel-{src}            | R (5)      | String         | The storage label                                     |
+
+1. Set's the LED status - generally "Off", "Low" or "High" (there may be others specific to your device like "AutoBrightnessAdjust")
+2. Set's the power savings mode - generally "Off", "Low" or "High" (there may be others specific to your device)
+3. Sending 'on' to this channel will reboot the device
+4. Sends an IRCC command to the device.
+This can either be the raw IRCC command (AAAAAwAAHFoAAAAYAw==) or can be a name (`Home`) that is transformed by the transformation file
+5.  These channels will be repeated by every storage source (ie source for a scheme of ```storage```).
+Example: if you have a ```USB1``` and ```CD``` storage sources, you'd have a ```st_uri-usb1``` and a ```st_uri-cd``` channel.
+Please note that, on many devices, the storage information is not reliable and a bit quirkly (the st_mounted status shows unmounted even though the storage is mounted).
+However, the st_mounted will reliably change when a source is physically mounted/unmounted from the unit.
+Just the initial status will likely be incorrect.
+
+### Video Channels (service ID of "video")
+
+The following list the channels for the video service.
+The video service allows management of the video (screen) itself.
+
+| Channel Type ID      | Read/Write | Item Type | Description                                         |
+| -------------------- | ---------- | --------- | --------------------------------------------------- |
+| audiosource          | RW         | String    | The audio source of the screen (speaker, headphone) |
+| bannermode           | RW         | String    | The banner mode (demo)                              |
+| multiscreenmode      | RW         | String    | The multiscreen mode (pip)                          |
+| pipsubscreenposition | RW         | String    | The pip screen position                             |
+| pipsubscreenposition | RW         | String    | The pip screen position                             |
```

Comment:
duplicate?

Reply:
Yep - should have been scene setting - fixed